macros/edr/tests/test_utils/get_test_type.sql

```diff
@@ -37,6 +40,8 @@
       {% do return("schema_change") %}
     {% elif flattened_test.short_name | lower in python_tests %}
         {% do return("python_test") %}
+    {% elif flattened_test.short_name | lower in dbt_tests %}
+        {% do return("dbt_test") %}
```

Comment:
I am not sure this is the right thing to do.
Maybe the definition of test type is a bit off, but I do not think that we need to categorize ore none anomaly / schema / python tests as `dbt test`.

Maybe we need to put it in a category like 'integrity` and start having a better types because dbt test doesn't says much and can be really confusing.

@haritamar WDYT?